custom_components/tuya_local/devices/Wada Energy Q8 Charger.yaml

```diff
@@ -0,0 +1,106 @@
+name: Q8 wall charger
```

Comment:
this top level name should be non-branded, in case other devices match the same config.
Suggest "EV charger" for the name.

Reply:
done, thanks